docker-compose.yaml

```diff
@@ -455,63 +455,204 @@ services:
       lagoon.rollout: daemonset
   harborclair:
     image: ${IMAGE_REPO:-lagoon}/harborclair
+    hostname: harborclair
```

Comment:
do we really need this hostname defined? in my understanding this is not needed, applies to all services

Reply:
From my research, this variable tells the created docker-container what its hostname should be. This is useful if a container needs to talk to itself. As far as I can tell, it shouldn't hurt anything to specify the container's hostname.